autoptsclient_common.py

```diff
@@ -1337,6 +1337,19 @@ def set_end():
     RUN_END = True
 
 
+def recover_at_exception(func):
+    def _recover_at_exception(*args, **kwargs):
+        while True:
+            try:
+                return func(*args, **kwargs)
+            except Exception as e:
+                logging.exception(e)
+                traceback.print_exc()
+                time.sleep(40)
```

Comment:
Magic number. Please define a global value with a descriptive name and a small comment if necessary. Just to avoid confusion and future questions about why do we need it and why this value was chosen.